### DISCUSSION TITLE
Option to set a different build folder?

### DISCUSSION THREAD

**Discussion by TypeTheory** [ocaml/ocaml-lsp#686] [Q&A] [answered]
I have a question. From what I understand ocaml-lsp-server/vscode extension examines the artifacts in _build folder in order to work. Is there an option of the vscode extension or ocaml-lsp-server, similar to dune's "--build-dir", to change with which build folder it works?

**Comment by ulugbekna** [comment, marked as answer]
Hi. Sorry, no, there’s no such option as far as I know.
